protocol/git/README.md

```diff
@@ -106,6 +106,7 @@ View a list of new commits. View changed files. Merge branch into master.
     git log origin/master..<branch-name>
     git diff --stat origin/master
     git checkout master
+    git pull
```

Comment:
Good questions @teoljungberg! Thanks for the feedback. I think even though `origin/master` is up to date, that it's still possible that your local `master` (which is what we're merging into) could be behind.

Good point about the rebase workflow. I think because we're merging `origin/master` into `master` it will always be a fast forward merge, unless something has gone wrong (i.e. we've got changes on our local master which haven't been pushed).

Also happy to close this PR though if we don't think it adds anything.